user-journey.adoc

```diff
@@ -0,0 +1,180 @@
+= User Journey for Istio/Jaeger
+
+ *Authors:* Lincoln Baxter, III and Charles Moulliard, Diógenes Rettori
+ *Version:* 0.6 - April 25, 2018
+
+This document describes the steps required to install a local Istio Openshift instance, necessary prerequisites (JDK, Maven, Launcher, Jaeger, …), and the operations to grant access for the user to the project. It also describes how the user can experiment with a mission selected using the launcher
```

Comment:
We need to clarify what the actual prerequisites are.  You mention JDK, Maven, Launcher, Jaeger as prerequisites in the introduction.  Which are actually prerequisites that the user has to install before they can install Istio? (Launcher and Jaeger are part of the installer, aren't they?)   Are some of these only required for zip flow or git flow?  And which version of OpenShift do you need?

Reply:
Only JDK, Maven are required if the user will git clone an istio mission, compile/test it locally. Jaeger or distributed Tracing is deployed on ocp using the `istiooc` installer. So it can be removed from this prereq line.